### PR TITLE
Current route in subquery

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: FrontBundle/Resources/config/subquery.yml [hunk 1]
```diff
@@ -14,6 +15,12 @@ services:
             - @request_stack
         tags:
             - { name: open_orchestra_front.sub_query.strategy }
+    open_orchestra_front.sub_query.current_route:
```
Comment (alavieille) [inline FrontBundle/Resources/config/subquery.yml:18]: where is used this strategy ?

Comment (nicolasThal) [inline FrontBundle/Resources/config/subquery.yml:18]: In the block with a form, you could need the current url for the action part!

Comment (alavieille) [inline FrontBundle/Resources/config/subquery.yml:18]: ok maybe a test for this strategy ?

Comment (nicolasThal) [inline FrontBundle/Resources/config/subquery.yml:18]: yes of course :)

**Comment by nicolasThal** [comment]
@itkg-alavieille pr updated

**Comment by alavieille** [comment]
:+1:

**Pull request merged**